modules/ducktests/tests/ignitetest/utils/ignite_test.py

```diff
@@ -26,9 +26,17 @@ class IgniteTest(Test):
     """
     Basic ignite test.
     """
+    TEMP_PATH_ROOT = "/mnt/ducktests"
```

Comment:
Python has tools to create temporary folders. You don't need this.

Reply:
Used in DiscoveryTest:     `NETFILTER_SAVED_SETTINGS = os.path.join(IgniteTest.TEMP_PATH_ROOT, "discovery_test", "netfilter.bak")`

Reply:
This is wrong usage, please, use appropriate tools. And revert this change to IgniteTest.

Reply:
Removed